src/test/scala/io/sdkman/repos/ApplicationRepoSpec.scala

```diff
@@ -40,11 +41,17 @@ class ApplicationRepoSpec
     }
   }
 
+  override def beforeAll() = {
```

Comment:
An expression, so can simply be something like this:
```suggestion
  override def beforeAll() = Mongo.startMongoDb()
```